acceptance/tests/resource/service/service_enable_linux.rb

```diff
@@ -119,10 +124,17 @@
     if is_sysV
       rc_symlinks = on(agent, "find /etc/ -name *#{package_name[platform]}", :accept_all_exit_codes => true).stdout
       # sles removes rc.d symlinks
-      if platform != 'sles'
+      if platform != 'sles' && (platform != 'debian' && majrelease != 8)
         (start_runlevels + kill_runlevels).each do |runlevel|
           assert_match("#{runlevel}.d/#{kill_symlink}", rc_symlinks, "did not find #{kill_symlink} in runlevel #{runlevel}")
         end
+      elsif platform == 'debian' && majrelease == 8
```

Comment:
If this comparison/block is ordered to come first, then we don't need two separate "is it deb8?" comparisons.